x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -643,6 +760,34 @@ export const ModelsList: FC<Props> = ({
           onTableChange={onTableChange}
           sorting={sorting}
           data-test-subj={isLoading ? 'mlModelsTable loading' : 'mlModelsTable loaded'}
+          childrenBetween={
+            isCalloutVisible ? (
+              <>
+                <EuiCallOut
```

Comment:
Looking at the suggestion from @mdefazio , shouldn't we only show this if ELSER v1 has already been downloaded? I am seeing it even though I have no V1 version of the model.

Reply:
Updated in https://github.com/elastic/kibana/pull/167407/commits/b478b9f88586934f5b2f73e830d03e2cdb446d11